optaplanner-persistence/optaplanner-persistence-jpa/pom.xml

```diff
@@ -79,16 +79,92 @@
       <artifactId>hibernate-entitymanager</artifactId>
       <scope>test</scope>
     </dependency>
+
+
     <dependency>
       <groupId>com.h2database</groupId>
       <artifactId>h2</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.codehaus.btm</groupId>
+      <artifactId>btm</artifactId>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Once we reuse the Persistence util from drools-persistence-jpa by importing it's test-jar as a dependency,
this btm dependency should be removed I think.